ui/marketplace/MarketplaceAppCard.tsx

```diff
@@ -92,93 +95,96 @@ const MarketplaceAppCard = ({
+                external={ external }
+                title={ title }
+                onClick={ onAppClick }
+              />
+              <MarketplaceAppIntegrationIcon external={ external } internalWallet={ internalWallet }/>
+            </Skeleton>
+
+            <Skeleton
+              isLoaded={ !isLoading }
+              color="text_secondary"
+              fontSize="xs"
+              lineHeight="16px"
             >
-              <Link
-                fontSize={{ base: 'xs', sm: 'sm' }}
-                fontWeight="500"
-                paddingRight={{ sm: 2 }}
-                href="#"
-                onClick={ handleInfoClick }
-              >
-                More info
-              </Link>
-            </Box>
-          ) }
+              <span>{ categoriesLabel }</span>
+            </Skeleton>
+          </Flex>
         </Flex>
 
-        <Flex
-          display={{ base: 'flex', sm: 'contents' }}
-          flexDirection="column"
-          gap={ 2 }
+        <Skeleton
+          isLoaded={ !isLoading }
+          fontSize="sm"
+          lineHeight="20px"
+          noOfLines={{ base: 2, md: 3 }}
         >
-          <Skeleton
-            isLoaded={ !isLoading }
-            marginBottom={{ base: 0, sm: 2 }}
-            fontSize={{ base: 'sm', sm: 'lg' }}
-            lineHeight={{ base: '20px', sm: '28px' }}
-            paddingRight={{ base: '25px', sm: 0 }}
-            fontWeight="semibold"
-            fontFamily="heading"
-            display="inline-block"
-            order={{ base: 'auto', sm: 2 }}
-          >
-            <MarketplaceAppCardLink
-              id={ id }
-              url={ url }
-              external={ external }
-              title={ title }
-              onClick={ onAppClick }
-            />
-            <MarketplaceAppIntegrationIcon external={ external } internalWallet={ internalWallet }/>
-          </Skeleton>
-
-          <Skeleton
-            isLoaded={ !isLoading }
-            marginBottom={{ base: 0, sm: 2 }}
-            color="text_secondary"
-            fontSize="xs"
-            lineHeight="16px"
-            order={{ base: 'auto', sm: 3 }}
-          >
-            <span>{ categoriesLabel }</span>
-          </Skeleton>
-
-          <Skeleton
-            isLoaded={ !isLoading }
-            fontSize={{ base: 'xs', sm: 'sm' }}
-            lineHeight="20px"
-            noOfLines={ 3 }
-            order={{ base: 'auto', sm: 4 }}
-          >
-            { shortDescription }
-          </Skeleton>
-        </Flex>
+          { shortDescription }
+        </Skeleton>
 
         { !isLoading && (
-          <IconButton
+          <Box
             display="flex"
             alignItems="center"
-            justifyContent="center"
+            justifyContent="space-between"
+            marginTop="auto"
+          >
+            <Link
+              fontSize="sm"
+              fontWeight="500"
+              paddingRight={{ md: 2 }}
+              href="#"
```

Comment:
I think that if you don't provide the `href` prop at all, it will not lead anywhere in the app. So, it is redundant here, no?

Reply:
It turned out that without `href` the styles were not applied correctly in this case. We decided to leave it as it is for now